### DISCUSSION TITLE
From #27 - is there a timeline for 'demo' accounts ? or some example price data in JSON

### DISCUSSION THREAD

**Discussion by GarthJL1965** [amberelectric/public-api#72] [Q&A]
Hi Myles & All 

I'm an (out of work) integration engineer who said to some gents on the Facebook "My Efficient Electric Home" forum I could probably help them with a way of switching a 240V circuit based on Amber 'price' (these guys are customers, not necessary technical) 

I was thinking with the right data, parsing etc, I could set them up to use use an IFTTT based switch, through IFTTT and or Zapier, possibly  Home Assistant (If it were for me I'd go a bit more direct and use an ESP8266/Shelly/.. based switch, but I'm not unleashing that on people who might electrocute themselves) - and there are plenty more ways to skin a cat here, but access to some data is the first requirement

I'm not an Amber customer myself - nor do I wish to cause pain in any form to Amber, so I'm happy to sign an NDA or such. 

So, is there any idea when/(if ?) I can get a 'demo' account/API Key/, or failing that, even a zip of some JSON data to poke around ? I'm happy to 'swap' - do some testing, documentation etc the last QA job I had was Ruby based, but I speak a little Python (Ssssss) 

cheers & thanks, stay safe all

**Comment by madpilot** [comment]
I don't have a timeline for demo accounts just yet, but you'll be able to view the docs (which has examples) without an account next week. Of if you are really impatient you can download the open API definitions here: https://app.amber.com.au/swagger.json - throw them in an online open API viewer and you'll can get started (eg https://editor.swagger.io)

Also, I already have a home assistant plug in which can save some time: https://github.com/madpilot/hass-amber-electric
